Makefile

```diff
@@ -198,6 +202,7 @@ prepare-build: prepare-build-init
 .PHONY: prepare
 prepare: prepare-build
 	mono --debug $(PREPARE_EXE) $(_PREPARE_ARGS)
+	msbuild $(BOOTSTRAP_MSBUILD_FLAGS) $(BOOTSTRAP_SOLUTION)
```

Comment:
Why do we need this change on *Windows*?

Reply:
This PR removes the "bootstrap" MSBuild tasks from the SLN, so we need `make prepare` to build them now.

I think it makes sense to build them at that step? or do you think `make all` should do that?